src/ReleaseHistory.md

```diff
@@ -1,4 +1,7 @@
 # SARIF Viewer Visual Studio extension Release History
+## **v2.1.19** [SARIF Viewer](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer)
+* BUGFIX: Fix tag highlighting (and selection in SARIF explorer window) being off by one column. [#218](https://github.com/microsoft/sarif-visualstudio-extension/issues/218)
```

Comment:
>column [](start = 89, length = 6)

Again, since we didn't publish 2.18 (yet) noone saw this, so no need to mention it.